README.md

```diff
@@ -26,44 +26,6 @@ Descheduler, based on its policy, finds pods that can be moved and evicts them.
 note, in current implementation, descheduler does not schedule replacement of evicted pods
 but relies on the default scheduler for that.
 
-Table of Contents
```

Comment:
I would prefer to keep the markdown TOC, at least until github's new feature is more widely known. Plaintext links work on all environments

Reply:
Something else we can do is point to the button in the readme. 

with regards to

> Plaintext links work on all environments

Can you think of an environment where the github native toc would not work?

Reply:
Ok, I had a look around other kubernetes-sigs projects to see what they are doing, and I gathered some that fall into the same place as us, so they have these characteristics: 
- kubernetes-sigs or k/ project
- Project is big enough
- Project does not have a website yet, so Readme is heavy/long enough

https://github.com/kubernetes-sigs/external-dns

https://github.com/kubernetes-sigs/metrics-server

https://github.com/kubernetes-sigs/kubefed

https://github.com/kubernetes-sigs/prometheus-adapter

https://github.com/kubernetes/kompose

:point_down:  The only two following above requirements with a TOC that I could find)

https://github.com/kubernetes/client-go 

https://github.com/kubernetes/kube-state-metrics

Reply:
external-dns has this funny thing "pointing" to the gh-toc:

![image](https://user-images.githubusercontent.com/2432275/216326145-e595b760-cea3-45a9-8e19-e6a750faf9bb.png)


Which I don't like, I think. Maybe writing something like "Click at the [bullet list icon] at the top left corner of the Readme visualization for the github generated table of contents"

Reply:
Thanks for checking what other projects are doing for reference, @knelasevero. Maybe it's just a personal preference of mine, so this isn't a blocker for me. We were talking about moving to a dedicated docs site anyway